### PR TITLE
Handle case branch indentation better

### PR DESCRIPTION
**Pull request by jmbockhorst**

### CODE REVIEW

File: test/codeActionTests/addMissingCaseBranches.test.ts [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+            ""
+
+        _ ->
+            Debug.todo "branch '_' not implemented"
+
+`;
+
+    await testCodeAction(
+      source,
+      [{ title: "Add missing case branches" }],
```
Comment (Razzeee) [inline test/codeActionTests/addMissingCaseBranches.test.ts:39]: I think we should do another pass later on and let the user know beforehand, if we're going to add a catch all pattern

Comment (jmbockhorst) [reply test/codeActionTests/addMissingCaseBranches.test.ts:39]: Yeah I was thinking we could do something better, but wasn't sure what.

**Comment by Razzeee** [comment]
It would be nice to cover ```
module Test exposing (..)


func a =
    case a of
```

But that's a parsing error and might be hard to do

**Comment by jmbockhorst** [comment]
That would be cool, but tricky to implement.

**Pull request merged**